sites/example-project/src/components/ui/CollapsibleSection.svelte

```diff
@@ -50,23 +50,24 @@
 			>
 		</button>
 		{#if folderList.filter((d) => d.folder === folder)[0].folderNameError}
-			<button class="folder-label nolink name-error" class:folder-selected={"/" + $page.path.split('/')[1] === folderHrefUri} aria-expanded={expanded} sveltekit:prefetch on:click={() => expanded = !expanded}>
+			<button>
+			<span class="folder-label nolink name-error" class:folder-selected={"/" + $page.url.pathname.split('/')[1] === folderHrefUri} aria-expanded={expanded} on:click={() => expanded = !expanded}>
 				<span class="alert-icon-folder">
 					<MdErrorOutline/>
 					<span class=info-msg>Folder names cannot include spaces. Use hyphens instead.</span>
 				</span>
 				{folderLabel}
 			</button>
 		{:else if folderList.filter((d) => d.folder === folder)[0].folderLink}
-			<a href={'/' + folder} aria-expanded={expanded} sveltekit:prefetch on:click={toggle}>
-                <div class=folder-label class:selected={$page.path === folderHrefUri} class:folder-selected={"/" + $page.path.split('/')[1] === folderHrefUri}>
+		<a href={'/' + folder} aria-expanded={expanded} on:click={toggle}>
+                <div class=folder-label class:selected={$page.url.pathname === folderHrefUri} class:folder-selected={"/" + $page.url.pathname.split('/')[1] === folderHrefUri}>
 				{folderLabel}
                 </div>
 			</a>
-		{:else}
-			<button class="folder-label nolink" class:folder-selected={"/" + $page.path.split('/')[1] === folderHrefUri} aria-expanded={expanded} sveltekit:prefetch on:click={() => expanded = !expanded}>
+		{:else}		
+			<span class="folder-label nolink" class:folder-selected={"/" + $page.url.pathname.split('/')[1] === folderHrefUri} aria-expanded={expanded} on:click={() => expanded = !expanded}>
```

Comment:
I wonder if the `button` to `span` might be some regression 🤔

Reply:
Good call, thanks